### PR TITLE
Harmony: Offline bar component

### LINKED ISSUE #806: Harmony: Offline bar component
Like in crew https://github.com/taskany-inc/crew/pull/446

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

- [x] Feature

## Related issues

Resolve #806

### CODE REVIEW

**Review by asabotovich — APPROVED** [review]

File: src/harmony/OfflineBanner/OfflineBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import { Alert } from '../Alert/Alert';
+
+import s from './OfflineBanner.module.css';
+
+interface OfflineBannerProps {
+    description: string;
+}
+
+export const OfflineBanner: React.FC<OfflineBannerProps> = ({ description }) => {
+    return <Alert view={'danger'} text={description} className={s.Alert} />;
```
Comment (awinogradov) [inline src/harmony/OfflineBanner/OfflineBanner.tsx:12]: Smbdy need help u with formatting ;)

**Review by awinogradov — CHANGES_REQUESTED** [review]

File: src/harmony/OfflineBanner/OfflineBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { Alert } from '../Alert/Alert';
+
+import s from './OfflineBanner.module.css';
+
+interface OfflineBannerProps {
+    description: string;
```
Comment (awinogradov) [inline src/harmony/OfflineBanner/OfflineBanner.tsx:8]: description > text

**Comment by Katrin-kudryash** [comment]
Untitled 12

**Review by awinogradov — APPROVED** [review]

**Review by tsumo — APPROVED** [review]

**Pull request merged**